consensus/polybft/checkpoint_manager_test.go

```diff
@@ -0,0 +1,265 @@
+package polybft
+
+import (
+	"errors"
+	"math/big"
+	"strconv"
+	"testing"
+
+	"github.com/0xPolygon/polygon-edge/consensus/ibft/signer"
+	"github.com/0xPolygon/polygon-edge/consensus/polybft/bitmap"
+	bls "github.com/0xPolygon/polygon-edge/consensus/polybft/signer"
+	"github.com/0xPolygon/polygon-edge/types"
```

Comment:
Nitpick but, is it worth it to separate local imports from external ones?

Reply:
VS Code auto-formatter formats it like this. I could manually change this file, but I'm sure there are more files accross entire codebase where is the same situation as here.

Reply:
If you separate them manually, goimport respects that separation, we'll later add this https://github.com/0xPolygon/go-ibft/pull/40/files#diff-6179837f7df53a6f05c522b6b7bb566d484d5465d9894fb04910dd08bb40dcc9R86-R87

Reply:
Fixed [8d0b4fb](https://github.com/0xPolygon/polygon-edge/pull/861/commits/8d0b4fb01e34b1cacf29839a2432a4768cf3fab0)